openshift/deploy/deploymentconfig/shipit.yml

```diff
@@ -108,18 +108,29 @@ objects:
                 requests:
                   cpu: 500m
                   memory: 256Mi
-              livenessProbe:
-                exec:
-                  command:
-                    - /usr/bin/env
-                    - bash
-                    - '-c'
-                    - '"ps ax | grep -v grep | grep sidekiq"'
+              ports:
+                - containerPort: 7433
+                  protocol: TCP
+              readinessProbe:
                 failureThreshold: 3
-                initialDelaySeconds: 120
+                httpGet:
+                  path: /
+                  port: 7433
+                  scheme: HTTP
+                initialDelaySeconds: 30
                 periodSeconds: 10
                 successThreshold: 1
                 timeoutSeconds: 1
+              livenessProbe:
+                failureThreshold: 3
+                httpGet:
+                  path: /
+                  port: 7433
+                  scheme: HTTP
+                initialDelaySeconds: 600
+                periodSeconds: 10
+                successThreshold: 1
+                timeoutSeconds: 30
```

Comment:
Do we want to add this to gracefully terminate? 

```
lifecycle:
        preStop:
          exec:
            # SIGTERM triggers a quick exit; gracefully terminate instead
            command: ["bundle", "exec", "sidekiqctl", "quiet"]
``` 
(from the [README](https://github.com/arturictus/sidekiq_alive))

Reply:
good call, I bet that's what's causing arturictus/sidekiq_alive#25